SU-SWS/stanford_basic
# Update decanter v6 and remove deprecated code.

# READY FOR REVIEW/

# Summary
- Updates deprecated code in latest v6 branch of decanter.

# Review By (Date)
- October 18th

# Urgency
- Med-High, cannot compile with the latest version unless deprecations have been changed. 

# Steps to Test

1. Check out this branch
2. Run `npm install`
3. Run `npm run build`
4. Run `npm run publish`
5. Review code changes

# Affected Projects or Products
- Everything this theme is doing.

# Associated Issues and/or People
- @pookmish  FYI
- @boznik  FYI

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
